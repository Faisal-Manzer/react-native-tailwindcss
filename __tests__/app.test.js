import React from 'react';
import ImportTailwind from './fixtures/ImportTailwind';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
    const tree = renderer.create(< ImportTailwind/>).toJSON();
    expect(tree).toMatchSnapshot();
});
