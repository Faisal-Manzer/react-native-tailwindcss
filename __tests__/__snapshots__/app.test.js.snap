// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 1,
      },
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "textAlign": "center",
        },
      ]
    }
  >
    The integration works
  </Text>
</View>
`;
